### PR TITLE
an attempt to clean the code

### PR DESCRIPTION
**Pull request by piyushmishra1416**

### CODE REVIEW

**Comment by piyushmishra1416** [comment]
fixes issue #1

File: src/components/select.css [hunk 1]
```diff
@@ -7,18 +7,14 @@
  
    width: 376px;
    height: 806px;
- 
-   /* Inside auto layout */
- 
+
```
Comment (ayush-AI) [inline src/components/select.css]: remove this extra space

File: src/components/select.css [hunk 2]
```diff
@@ -7,18 +7,14 @@
...
    flex: none;
    order: 1;
    flex-grow: 0;
  }
  .toast {
    box-sizing: border-box;
```
Comment (ayush-AI) [inline src/components/select.css]: here also

Comment (ayush-AI) [reply src/components/select.css]: You can use prettier to do the work for you

File: src/components/sidepanel.css [hunk 3]
```diff
@@ -338,56 +291,23 @@ Border — Table
 border-top: 1px solid #CECECE;
 
 }
-.Frame21948 {
+/* .search-box {
```
Comment (ayush-AI) [inline src/components/sidepanel.css:246]: do we need this "/*" here? if not then make it to this
```suggestion
 .search-box {
```

**Review by ayush-AI — CHANGES_REQUESTED** [review]
Please improve the lints using prettier.

File: src/App.tsx
Comment (ayush-AI) [inline src/App.tsx:1]: improve the lints in this file

**Review by ayush-AI — CHANGES_REQUESTED** [review]

**Pull request merged**